core/src/test/scala/com/evolutiongaming/catshelper/LogSpec.scala

```diff
@@ -78,6 +78,16 @@ class LogSpec extends AnyFunSuite with Matchers {
 
     io.unsafeRunSync() shouldEqual null
   }
+
+  test("logback implementation") {
+    val io = for {
+      logOf <- LogOf.logback[IO]
+      log <- logOf(getClass)
+      _ <- log.info("hello from logback", Log.Mdc("k" -> "test value for K"))
```

Comment:
what are we testing here? :)

Reply:
possibility to log message and see it as output. Do you know better way to test such logger?

Reply:
register custom appenders from code in test with vars and check those are assigned to correct values?